devops/aws-codebuild/Jenkinsfile.cd

```diff
@@ -22,12 +32,11 @@ pipelineWrapper({
             checkout scm
         }
 
-        def evernymRepo = evLibrary.EvernymRepo.new(this)
-        def sovrinRepo = evLibrary.SovrinRepo.new(this)
-        def utils = evLibrary.Utils.new(this)
-        def buildCtx = evLibrary.AwsCodeBuildHelper.BuildCtx.new('plugin',
+        def sovrinRepo = sovLibrary.SovrinRepo.new(this)
+        def utils = sovLibrary.Utils.new(this)
+        def buildCtx = sovLibrary.AwsCodeBuildHelper.BuildCtx.new('plugin',
             ['devops', 'sovtoken/sovtoken/__metadata__.py', 'sovtokenfees/sovtokenfees/__metadata__.py'])
-        def awsCBHelper = evLibrary.AwsCodeBuildHelper.new(this, buildCtx)
+        def awsCBHelper = sovLibrary.AwsCodeBuildHelper.new(this, buildCtx)
 
 
```

Comment:
The code below hasn't been changed but it's not appropriate for public build server since all jobs are run on Jenkins master. Need to specify labels to use agents instead.